cookbooks/providers/application/providers/default.rb

```diff
@@ -1,5 +1,26 @@
 action :install do
 
+  if new_resource.letsencrypt
+    log %{Install certbot-auto...}
+    execute 'wget certbot-auto' do
+      user 'root'
+      command 'wget https://dl.eff.org/certbot-auto -O /root/certbot-auto && chmod a+x /root/certbot-auto && mv /root/certbot-auto /usr/local/bin/certbot-auto'
+    end
+
+    log %{Update certbot-auto...}
+    execute 'update certbot-auto' do
+      user 'root'
```

Comment:
wherever we use `user`, I believe we should specify `group` as well. That would be `'root'` then as well I think.